app/build.gradle

```diff
@@ -152,7 +145,7 @@ dependencies {
 
     systemImplementation "org.mozilla.components:browser-engine-system:$moz_components_version"
     // Required to override android.support lib in GeckoView dependency (uses palette-v7:26.1.0)
-    geckoImplementation "com.android.support:palette-v7:$support_libraries_version"
+    geckoImplementation "androidx.palette:palette:$support_libraries_version"
```

Comment:
Did you check gecko variant's correctly rendering web pages? :)

Reply:
I hadn't!  Seems to work ok, aside from the known issues.  Is there anything in particular you would like me to look at?